src/app/PlaceDetailsData.js

```diff
@@ -130,7 +130,9 @@ const PlaceDetailsData: DataTableEntry<PlaceDetailsProps> = {
 
       const equipmentPromise = equipmentInfoId ? fetchEquipment(equipmentInfoId, useCache) : null;
       const equipmentInfo = equipmentPromise
-        ? isServer ? await equipmentPromise : equipmentPromise
```

Comment:
I think nested ternary operators are a bit difficult to understand – how about this? :)

```javascript
(isServer ? await equipmentPromise : equipmentPromise) || null
```